src/ui/application-window.vala

```diff
@@ -268,6 +294,34 @@ namespace Peek.Ui {
         }
       }
     }
+    
+    
+    
+
+    [GtkCallback]
+    private void on_gif_button_clicked (Button source) {
+    	recorder.output_format="gif";
```

Comment:
I broke this in master branch by adding a recorder.config object. You should rebase this branch against master and change this to `recorder..config.output_format`. Btw, that's a nice example of there being no merge conflict, but built will fail anyway ;)